shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -215,8 +220,6 @@ public void detachFromRenderer() {
         disconnectSurfaceFromRenderer();
       }
 
-      pause();
```

Comment:
Why are you removing the calls to pause() from detach?

Reply:
The call to pause() set `isPaused` to `true`. 
```java
  public void pause() {
    if (flutterRenderer == null) {
      Log.w(TAG, "pause() invoked when no FlutterRenderer was attached.");
      return;
    }
    isPaused = true;
  }
```

Having `isPaused` set to `true` prevents the `startRenderingToSurface` function from creating a new surface.


```java
  // FlutterRenderer and getSurfaceTexture() must both be non-null.
  private void connectSurfaceToRenderer() {

    ...

    flutterRenderer.startRenderingToSurface(getHolder().getSurface(), isPaused);
  }
```

```java
  public void startRenderingToSurface(@NonNull Surface surface, boolean onlySwap) {
    ...

    this.surface = surface;

    if (onlySwap) {
      // In the swap case we are just swapping the surface that we render to.
      flutterJNI.onSurfaceWindowChanged(surface);
    } else {
      // In the non-swap case we are creating a new surface to render to.
      flutterJNI.onSurfaceCreated(surface);
    }
  }
```

Reply:
What I'm asking you to do is fix the root bug and not just stop calling pause when it should be called.

Reply:
> What I'm asking you to do is fix the root bug and not just stop calling pause when it should be called.

I believe that I am **INDEED** addressing the root bug.

As stated in the comments of the `detachFromRenderer` function, if the `detachFromRenderer` function is called, it indicates that the `RenderSurface` is no longer needed and its underlying surface will also be detached. 

In my view, calling `pause()` within `detachFromRenderer()` is superfluous and contributes nothing to the problem that https://github.com/flutter/engine/pull/47358 aims to solve.

I completely fail to understand why you insist on retaining `pause` in `detachFromRenderer`.


```java
  /**
   * Invoked by the owner of this {@code FlutterSurfaceView} when it no longer wants to render a
   * Flutter UI to this {@code FlutterSurfaceView}.
   *
   * <p>This method will cease any on-going rendering from Flutter to this {@code
   * FlutterSurfaceView}.
   */
  public void detachFromRenderer() {
    ...
  }
```